### PR TITLE
Fixed broken tests

### PR DESCRIPTION
**Pull request by torcellite**

### CODE REVIEW

**Comment by torcellite** [comment]
Not sure why the tests aren't being loaded. Seem to work fine on my end.

screen shot 2015-06-20 at 1 12 53 pm

File: test/basic.html [hunk 1]
```diff
@@ -46,31 +46,29 @@
...
+        MockInteractions.tap(r1);
       });
 
       test('toggle button via click', function() {
         r1.checked = true;
         r1.addEventListener('click', function() {
-          assert.isFalse(r1.getAttribute('aria-checked'));
+          assert.isFalse(r1.getAttribute('aria-checked') != 'false');
           assert.isFalse(r1.checked);
-          done();
```
Comment (notwaldorf) [inline test/basic.html:61]: This is incorrect, and I think it accidentally got broken and never fixed :(. You should keep the `done();` statement here, and add it as a parameter to the test (i.e. in L47), etc.

**Comment by notwaldorf** [comment]
@torcellite Thanks for the fix! Would you mind taking a look at the comment I left, please? :)

**Comment by notwaldorf** [comment]
Closing this since it's super old, and this code has already been fixed on master

**Pull request closed without merging**